spec/features/create_new_work_spec.rb

```diff
@@ -130,6 +130,11 @@
       expect(page).to have_content('2020-03-06/2020-10-30')
       expect(page).to have_content('Whatever')
       expect(page).to have_content('CC-PDDC Public Domain Dedication and Certification')
+
+      visit dashboard_path
+
+      # We should not see the delete button for this work since it is not a draft
+      expect(page).not_to have_selector("#remove-work-#{Work.last.id}")
```

Comment:
Can you do this by querying on the aria-label rather than the id? The aria-label is more important to the users. The id serves no purpose that I can tell.

Reply:
Done